test/workers/test_virtual.py

```diff
@@ -34,10 +37,14 @@ def test_send_msg():
     obj_id = obj.id
 
     # Send data to bob
+    start_time = time()
     me.send_msg(ObjectMessage(obj), bob)
+    elapsed_time = time() - start_time
 
     # ensure that object is now on bob's machine
     assert obj_id in bob._objects
+    # ensure that object was sent 0.1 secs later
+    assert abs(elapsed_time - me.message_pending_time) < 0.1
```

Comment:
That’s confusing why not just ensure rlapsed_time > 0.1 ?

Reply:
You are right, will do that.

Reply:
Done